### PR TITLE
Feature/css work

### PR DESCRIPTION
**Pull request by john-lennie**

### CODE REVIEW

File: app/views/projects/_partial_project_anonymous.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<% @projects.each do |project| %>
+  <% if project.mentor_id && project.mentee_id && project.public %>
+  <% mentor = User.get_user(project.mentor_id)%>
+  <% mentee = User.get_user(project.mentee_id)%>
+    <section class="mentor-project">
+      <section class="users">
+        <section class="user">
+          <h5>Mentor:</h5>
+          <div class="user-image"><%= image_tag mentor.avatar.url(:thumb) %></div>
+          <p><%= mentee.name %></p>
```
Comment (1andee) [inline app/views/projects/_partial_project_anonymous.html.erb]: This should be mentor.name

Comment (1andee) [inline app/views/projects/_partial_project_anonymous.html.erb:10]: This should be mentor.name instead of mentee.name

Comment (john-lennie) [reply app/views/projects/_partial_project_anonymous.html.erb:10]: thanks

**Comment by 1andee** [comment]
Looks beautiful!!!

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -189,11 +196,13 @@ main {
...
+}
+
 .user-image {
-  img {
-    max-width: 100px;
-    margin: 0 0 15px;
-  }
+  max-width: 100px;
+  margin: 0 0 15px;
 }
```
Comment (1andee) [inline app/assets/stylesheets/scaffolds.scss:206]: Changes to `.user-image` class removed `img` width constraints. Element should be constrained to 100px for proper scaling of `/images/missing.png`

See http://imgur.com/a/6t53E

**Pull request merged**